app/models/metasploit/credential/krb_ticket.rb

```diff
@@ -0,0 +1,132 @@
+require 'json'
+require 'base64'
+
+# A {Metasploit::Credential::Private} This class contains information relevant to a Kerberos Ticket
+# https://www.rfc-editor.org/rfc/rfc4120.html#section-5.3
+#
+# The data format used for the ticket is a MIT krb5 ccache, although it is treated as an opaque base64 blob string
+#
+# {#data} is a serialized ruby hash with the keys defined in {Metasploit::Credential::KrbEncKey::ACCEPTABLE_DATA_ATTRIBUTES}
+class Metasploit::Credential::KrbTicket < Metasploit::Credential::Private
```

Comment:
In framework we seem to be using the prefix of `Krb5` rather than `Krb` should we keep that consistent here too?